packaging/debian-sid/patches/0007-i18n-use-dummy-localizations-to-avoid-dependencies.patch

```diff
@@ -135,14 +135,13 @@ index 352e9a65d3c9ff802c18c8dbac613668aa23064d..12885f7149ff005e189762ae93f7eb98
 +	}
  }
 diff --git a/i18n/i18n_test.go b/i18n/i18n_test.go
-index 81cb050a76b824d1b83e97e02c08628520329d96..86b59f3b439e9c27d37cfb35e21d84724c4cd52f 100644
+index b8b0b1ea57..86b59f3b43 100644
 --- a/i18n/i18n_test.go
 +++ b/i18n/i18n_test.go
-@@ -20,143 +20,28 @@
+@@ -20,142 +20,28 @@
  package i18n
  
  import (
--	"io/ioutil"
```

Comment:
So the problem was this import being unused?

Reply:
No, the patch wouldn't apply because this import wasn't in the original file any more.

Reply:
Ahhh I see. Thanks for clarifying!